src/api/core/componentStorage.ts

```diff
@@ -1,6 +1,19 @@
 // tslint:disable:max-classes-per-file
 import { Tokens } from "./tokenManager";
 
+/**
+ * fetch an arrau of all aliases for one access token
+ * @internal
+ */
+export const getAliases = (accessToken: string, tokens: tokenList): string[] => Object.entries(tokens)
```

Comment:
What is an 'alias'? When I look at this function it looks like you just fetch the 'full' token from the list of available tokens using the provided accessToken?

Reply:
You can set multiple aliases per token as you can see here --> https://github.com/jexia/jexia-sdk-js/blob/master/src/api/core/tokenManager.ts#L184-L194

As you can see, there is a filter that fetch an alias names that match the access_token. 

```
.filter(([, token]) => token.access_token === accessToken)
```

See also https://docs.jexia.com/ums/#sign-in-a-user for the use of alias ;-)